src/components/atoms/Toggler/Toggler.tsx

```diff
@@ -1,24 +1,8 @@
-import React, { ChangeEventHandler } from "react";
+import React from "react";
 
 import { Checkbox, CheckboxProps } from "components/atoms/Checkbox";
 
-export interface TogglerProps extends CheckboxProps {
-  defaultToggled: boolean;
-  onToggle: ChangeEventHandler<HTMLInputElement>;
-  containerClassName?: string;
-}
-
-export const Toggler: React.FC<TogglerProps> = ({
-  defaultToggled,
-  onToggle,
-  containerClassName,
+export const Toggler: React.FC<CheckboxProps> = ({
+  toggler = true,
```

Comment:
No need for `toggler` to be a prop here at all.
```suggestion
export const Toggler: React.FC<CheckboxProps> = ({
```

Reply:
If this prop gets moved to the place `<Toggler>` is used, I can basically delete the entire component. But I'm still holding on to the hope some` made it so it can be added with own style in future.

Reply:
I suggest this OOS until the idea behind adding Toggler is examined later on.

Reply:
See https://doesitsparkle.slack.com/archives/C0229C1EC86/p1625921088289300?thread_ts=1625912773.269100&cid=C0229C1EC86

We absolutely want to keep `Toggler` and some of your other 'remove togglers props' changes here are probably undesirable as well.

Reply:
My suggestion is separate PR that re-examines `Toggler`'s usues and based on that, have own props that don't extend from CheckboxProps so that there's no ambiguities w.r.t. `defaultChecked`/`defaultToggled` and similar properties.

 An encapsulation, less leaky abstraction.

Reply:
I'll fix this up in this PR

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1775/commits/499e6b522507d0bbfdc29e110e0fddd441de1af4